### PR TITLE
doc: fixed broken urls

### PR DESCRIPTION
**Pull request by falcon78921**
- [ ] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

Signed-off-by: James McClune jmcclune@mcclunetechnologies.net

### CODE REVIEW

File: doc/rados/operations/control.rst [hunk 1]
```diff
@@ -100,7 +100,7 @@ Write a copy of the most recent OSD map to a file. See
 
 	ceph osd getmap -o file
 
-.. _osdmaptool: ../../man/8/osdmaptool
+.. _osdmaptool: http://docs.ceph.com/docs/master/man/8/osdmaptool/
```
Comment (tchaikov) [inline doc/rados/operations/control.rst]: it should be
```rst
.. _osdmaptool: /man/8/osdmaptool
```

or better off using the `:ref:` directive as @alfredodeza suggests.

Comment (falcon78921) [reply doc/rados/operations/control.rst]: @tchaikov I added a ``reST`` label for ``osdmaptool`` (I didn't see a reference label above the section title). 

This should work. If I missed something, please let me know.

**Comment by falcon78921** [comment]
jenkins retest this please

**Comment by falcon78921** [comment]
jenkins render docs

**Comment by tchaikov** [comment]
```
stderr: fatal: unable to access 'https://github.com/ceph/ceph/': Failed to connect to github.com port 443: No route to host
```

jenkins render docs again, if you please.

**Comment by tchaikov** [comment]
jenkins render docs

**Comment by tchaikov** [comment]
this breaks the build, see https://shaman.ceph.com/builds/ceph/master/4cf9e35cf5a3e7440651ec0634fd572b972fa0f6/  

a fix is posted at #23580

**Comment by falcon78921** [comment]
Thanks! :) @tchaikov

**Pull request merged**